parsers/AR.py

```diff
@@ -143,8 +158,50 @@ def fetch_exchange(
     logger: Logger = getLogger(__name__),
 ) -> dict:
     """Requests the last known power exchange (in MW) between two zones."""
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates")
+
+    sorted_zone_keys = sorted([zone_key1, zone_key2])
+    sorted_codes = "->".join(sorted_zone_keys)
+    flow: Union[float, None] = None
+    target_datetime = arrow.now(tz="America/Argentina/Buenos_Aires").datetime
+
+    if sorted_codes in SUPPORTED_EXCHANGES:
+        current_session = session or Session()
+
+        api_cammesa_response = current_session.get(CAMMESA_EXCHANGE_ENDPOINT)
+        assert api_cammesa_response.status_code == 200, (
+            "Exception when fetching exchange for "
+            "{}: error when calling url={}".format(
+                sorted_codes, CAMMESA_EXCHANGE_ENDPOINT
+            )
+        )
```

Comment:
This should be using `f"string {object}"` instead both for readability and preformance reasons.